app/services/vot/supported_component_values.rb

```diff
@@ -38,6 +38,12 @@ module SupportedComponentValues
       implied_component_values: [P1],
       requirements: [:biometric_comparison],
     ).freeze
+    Pe = ComponentValue.new(
```

Comment:
what does `Pe` mean here?

Reply:
"Proofing Enhanced?" 😆  I really put it out there for feedback. I initially had it as "P2" since it was a step up from the `P1` component. I'm really happy to change it to whatever is more intuitive though.

Reply:
ahh ok, I wonder what the convention was here? It isn't obvious to me what any of the other names in the file mean either.

Reply:
I don't think that `P2` is the way to go. This isn't necessarily as step up from the base proofing experience since it is scoped to in-person. For the same reason proofing with a biometric was named `Pb` instead of `P2`.

The naming convention for vectors of trust is described in [RFC 8485]. They are an upper-case letter followed by a lower-case letter or a number.